styles/mobile-styles.css

```diff
@@ -275,4 +319,16 @@
     flex-direction: column;
     align-items: center;
   }
+
+  .mobile-menu {
+    background-color: #fff;
+    width: 100vw;
+    height: 100vh;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 44px;
+    position: fixed;
+  }
```

Comment:
- [ ] Ouch this is close but the mobile menu as shown in the Figma design doesn't fill the entire screen it has some gap at the bottom with rounded corners so kindly update your design to match it.
- Figma design:
  ![image](https://github.com/BHObserver/my-portfolio/assets/104968236/a5411f89-f358-4c60-85a8-d7233db90628)
- Your design:
  ![image](https://github.com/BHObserver/my-portfolio/assets/104968236/3f491f7a-0e0d-4fd1-bbdb-0276f47ae929)